README.md

```diff
@@ -1,2 +1,97 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # TO-DO-LIST
-It simply lists the things that you need to do and allows you to mark them as complete, and it uses ES6 and Webpack.
+
+> Description the project.
+- It simply lists the things that you need to do and allows you to mark them as complete, and it uses ES6 and Webpack.
+- Single page website.
+- configure with linters.
+
+## Screenshot
+
+![readme](./assets/screenshoots/M1.png)
+![readme](./assets/screenshoots/M2.png)
+![readme](./assets/screenshoots/D1.png)
+![readme](./assets/screenshoots/D2.png)
+
+# Live Demo
+
+[Live Demo Link](https://reemmohamedabdelfatah.github.io/TO-DO-LIST/)
```

Comment:
- [ ] Your live demo link is broken :cry: Please fix it in order to make it available and your readme more professional :+1: